source/js/components/join/join.jsx

```diff
@@ -53,7 +53,7 @@ export default class JoinUs extends React.Component {
     });
 
     let signupState = classNames({
-      'row py-3 py-md-4 px-md-2': true,
+      'row py-5 py-md-5 px-md-2': true,
```

Comment:
Assuming this change is indeed supposed to affect mobile and medium+, the `py-md-5` can just go away.